page/ajax/working-with-jsonp.md

```diff
@@ -5,30 +5,30 @@
 	"attribution": [ "jQuery Fundamentals" ]
 }</script>
 
-The advent of JSONP — essentially a consensual cross-site scripting hack — has opened the door to powerful mashups of content. Many prominent sites provide JSONP services, allowing you access to their content via a predefined API. A particularly great source of JSONP-formatted data is the [Yahoo! Query Language](http://developer.yahoo.com/yql/console/), which we'll use in the following example to fetch news about cats.
+The advent of JSONP — essentially a consensual cross-site scripting hack — has opened the door to powerful mashups of content. Many prominent sites provide JSONP services, allowing you access to their content via a predefined API. A particularly great source of JSONP-formatted data is the [Wikipedia API](https://en.wikipedia.org/w/api.php?action=help&modules=query), which we'll use in the following example to fetch a list of articles about [JavaScript libraries](https://en.wikipedia.org/wiki/Category:Free_software_programmed_in_JavaScript).
 
 ```
-// Using YQL and JSONP
 $.ajax({
-	url: "http://query.yahooapis.com/v1/public/yql",
+	// Tell MediaWiki what we want and that we want JSON
+	url: "https://en.wikipedia.org/w/api.php",
+	data: {
+		format: "json",
+		action: "query",
+		list: "categorymembers",
+		cmtitle: "Category:Free_software_programmed_in_JavaScript",
+		cmprop: "title",
+		cmlimit: 500
+	},
 
-	// The name of the callback parameter, as specified by the YQL service
+	// Name of the query parameter that jQuery will add, to set the callback function
```

Comment:
The comma seems misaligned to me here. Should it just be removed?

Reply:
Agreed. Comma can be removed.